app/actions.py

```diff
@@ -629,10 +629,40 @@ def cursorSelectPageUp(self):
     Returns:
       None.
     """
-    if self.selectionMode == app.selectable.kSelectionNone:
-      self.doSelectionMode(app.selectable.kSelectionCharacter)
+    if self.selectionMode != app.selectable.kSelectionCharacter:
+      self.selectionCharacter()
+    self.__cursorPageUp()
+
+  def cursorAltSelectPageDown(self):
+    """
+    DESCRIPTION HERE
+
+    Args:
+      None.
+
+    Returns:
+      None.
+    """
+    if self.selectionMode != app.selectable.kSelectionBlock:
+      self.selectionBlock()
+    self.__cursorPageDown()
+
+  def cursorAltSelectPageUp(self):
+    """
+    DESCRIPTION HERE
+
+    Args:
+      None.
+
+    Returns:
+      None.
+    """
```

Comment:
With a name like cursorSelectBlockPageUp() for a function that calls
Selection Block and cursor PageUp, can the doc string be left off.

Reply:
It does have a different behavior though, so we should probably document its function. Just don't know if how kSelectionBlock's implementation is making the effect you want from the function.

Reply:
Edit: it behaves like the GUI editor so may I'll just leave it off then. I'll update the cursorSelectPageUp/Down docs too